units/systemd-time-wait-sync.service.in

```diff
@@ -29,7 +29,6 @@ Conflicts=shutdown.target
 [Service]
 Type=oneshot
 ExecStart={{ROOTLIBEXECDIR}}/systemd-time-wait-sync
-TimeoutStartSec=infinity
```

Comment:
hmm, this defeats the point of this service. it's an optional service (you have to explicitly enable) if you *want* to delay the boot process like this. if we then end up delaying it, why bother at all?

this smells as a specialist change for some people who want some middle ground of "yeah i really want correct time, except when i don't". But we cannot know if people want that.

It sound to me as if people should configure this locally, as they want, via a simple drop-in. it's impossible for us to know what is right here.